pkg/client/results/reader.go

```diff
@@ -330,7 +331,7 @@ func ConfigFile(version string) string {
 	case VersionEight:
 		return "config.json"
 	default:
-		return "meta/config.json"
+		return path.Join("meta", "config.json")
```

Comment:
So this is one of the things that wasn't well flushed out because it didn't have to be in the past since the server was always on linux nodes.

The git commit mentions the logic here:
 - on the client and server, when dealing with their own files you need to use filepath
 - when doing anything with files in the tarball, use path.

This way all local file handling works just fine (filepath) but the files in the tarball are in a predictable format (path) for traversing. Otherwise it gets messy to (1) record/detect what OS was responsible for making the tar (2) modifying which path separator to use when traversing